handlers/http.go

```diff
@@ -46,12 +51,17 @@ func (hrl httpRateLimiter) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	matches, err := hrl.rateLimiter.Add(request)
 	if err != nil && err != leakybucket.ErrorFull {
 		log.Printf("[%s] ERROR: %s", guid, err)
-		w.WriteHeader(500)
-		return
+		if !hrl.AllowOnError {
+			log.Printf("[%s] WARNING: bypassing rate limiter due to Error")
```

Comment:
is it worth making this a metric so we can alert on it? or do we have another mechanism we're happy with for determining that redis is down?